lib/diagnostic.js

```diff
@@ -15,7 +15,20 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
-
+  db.createCollection(houses {
```

Comment:
A collection can be created by inserting documents into it even if it doesn't already exist.  Here are two solutions to this question.

```javascript
db.houses.insert({
  name: 'House Arryn',
  motto: 'As High as Honor'
});

db.houses.insert({
  name: 'House Stark',
  motto: 'Winter is Coming'
});

db.houses.insert({
  name: 'House Targaryen',
  motto: 'Fire and Blood'
});
```

or

```javascript
db.houses.insert([{
  name: 'House Arryn',
  motto: 'As High as Honor'
}, {
  name: 'House Stark',
  motto: 'Winter is Coming'
}, {
  name: 'House Targaryen',
  motto: 'Fire and Blood'
}]);
```